concourse/tasks/bosh_cancel_all_tasks/task.yml

```diff
@@ -0,0 +1,19 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/bosh-cli-v2
+    tag: c88f3e0b03558c987693fad3f180d9052b77342c
```

Comment:
Why a new `bosh_cli_v2` version is required ? Or maybe we need to update to latest version.

Reply:
This one has ruby in it.